scripts/config/msu/string.nut

```diff
@@ -26,4 +26,14 @@
 		}
 		return true;
 	}
+
+	function startsWith( _string, _start )
```

Comment:
This seems like a mistake i.e. changes to the strings utils file in this PR. Please remove the changes to this file.

Reply:
This was caused by the automatic base change, fixed by rebasing